src/navsat_transform.cpp

```diff
@@ -411,8 +413,9 @@ namespace RobotLocalization
     }
     else
     {
-      std::string utm_zone_tmp;
-      NavsatConversions::LLtoUTM(latitude, longitude, cartesian_y, cartesian_x, utm_zone_tmp);
+      int zone_tmp;
+      bool nortp_tmp;
+      GeographicLib::UTMUPS::Forward(latitude, longitude, zone_tmp, nortp_tmp, cartesian_x, cartesian_y, utm_zone_);
```

Comment:
Why can't we keep using `LLtoUTM` here? Doesn't your other PR just wrap to the GeographicLib call anyway?

Reply:
Sorry for the confusion.

I started on the other PR. But since that makes the whole conversion functions almost empty, one might as well use the functions directly, removing all custom functions. Actually I would want to remove the whole conversions file in this PR. But I don't know who might be using it.

The reason that this is also needed is because the library uses a boolean `northp` instead of string parsing the `utm_zone_` variable. It's much easier to track the boolean than to keep the information in a string and encode/decode that.

Reply:
OK, sounds good. As I said in the other PR, we'll remove that file for ROS2 Galactic onward.